### PR TITLE
test

### PR DESCRIPTION
**Pull request by chaines**
testing pull request

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+asdkfja;lksdjfa dsklf;haej nljvka d;lkfasfadfasdfasdasdf adklaj;glkjda; flkajdf
+asdkfja;lksdjfa dsklf;haej nljvka d;lkfasfadfasdfasdasdf adklaj;glkjda; flkajdf
+asdkfja;lksdjfa dsklf;haej nljvka d;lkfasfadfasdfasdasdf adklaj;glkjda; flkajdf
+asdkfja;lksdjfa dsklf;haej nljvka d;lkfasfadfasdfasdasdf adklaj;glkjda; flkajdf
+asdkfja;lksdjfa dsklf;haej nljvka d;lkfasfadfasdfasdasdf adklaj;glkjda; flkajdf
+asdkfja;lksdjfa dsklf;haej nljvka d;lkfasfadfasdfasdasdf adklaj;glkjda; flkajdf
+asdkfja;lksdjfa dsklf;haej nljvka d;lkfasfadfasdfasdasdf adklaj;glkjda; flkajdf
+asdkfja;lksdjfa dsklf;haej nljvka d;lkfasfadfasdfasdasdf adklaj;glkjda; flkajdf
+asdkfja;lksdjfa dsklf;haej nljvka d;lkfasfadfasdfasdasdf adklaj;glkjda; flkajdf
+asdkfja;lksdjfa dsklf;haej nljvka d;lkfasfadfasdfasdasdf adklaj;glkjda; flkajdf
```
Comment (chaines) [inline CONTRIBUTING.md:11]: this line looks redundant

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
```
Comment (Hkanabar) [inline CONTRIBUTING.md:25]: Test comment

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+adfasdfasdasdf adklaj;glkjda; flkajdf
+asdkfja;lksdjfa dsklf;haej nljvka d;lkfasfadfasdfasdasdf adklaj;glkjda; flkajdf
+asdkfja;lksdjfa dsklf;haej nljvka d;lkfasfadfasdfasdasdf adklaj;glkjda; flkajdf
+asdkfja;lksdjfa dsklf;haej nljvka d;lkfasfadfasdfasdasdf adklaj;glkjda; flkajdf
+asdkfja;lksdjfa dsklf;haej nljvka d;lkfasfadfasdfasdasdf adklaj;glkjda; flkajdf
+asdkfja;lksdjfa dsklf;haej nljvka d;lkfasfadfasdfasdasdf adklaj;glkjda; flkajdf
+asdkfja;lksdjfa dsklf;haej nljvka d;lkfasfadfasdfasdasdf adklaj;glkjda; flkajdf
+asdkfja;lksdjfa dsklf;haej nljvka d;lkfasfadfasdfasdasdf adklaj;glkjda; flkajdf
+asdkfja;lksdjfa dsklf;haej nljvka d;lkfasfadfasdfasdasdf adklaj;glkjda; flkajdf
+asdkfja;lksdjfa dsklf;haej nljvka d;lkfasfadfasdfasdasdf adklaj;glkjda; flkajdf
```
Comment (Katerina-spb) [inline CONTRIBUTING.md:10]: testtest

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+People be mad at me
```
Comment (chaines) [inline CONTRIBUTING.md:29]: much better :)

**Comment by chaines** [comment]
This code was trash. Who wrote this garbage

**Pull request closed without merging**